### DISCUSSION TITLE
File drag and drop with eframe + epi

### DISCUSSION THREAD

**Discussion by Ell** [emilk/egui#744] [Q&A]
I'm trying to get file drag and drop working (desktop file -> app window) and none of the events seem to be firing off. I copied the example from https://github.com/emilk/egui/blob/a256ca115bd052a87d1b6c7f83f8a9bcccf088ef/egui_demo_lib/src/wrap_app.rs#L152

My current test:
https://gist.github.com/Ell/09d2a3875375a2c4dc9cba919f382541

Am I missing something obvious?

**Comment by emilk** [comment]
That does look right to me. Does dropping files into `egui_demo_app` work (`git clone egui && cd egui && cargo run`)?

If not, perhaps winit doesn't support file drag-and-drop on your platform

**Reply by Ell** [reply]
I think this is an egui issue, a quick test with winit: https://gist.github.com/Ell/b5f66e74820e8cb86450d7c2b10c8838
seems to cause intended behavior (everything prints fine)

I cannot get the demo to detect my file drag and drop 

Im on windows 10 using rust nightly and pulling egui from git

**Reply by Ell** [reply]
On further investigation it seems like egui is blocking the window from being able to see the file or something. I set a breakpoint on all file drop related events and none of them get fired in egui when dragging and dropping a file into a display setup with egui. All other events trigger fine (tested keyboard and mouse inputs, all trigger breakpoints).  I'm not really sure what part of the code to look at to dig into this more but I'd love to help if someone has pointers.
